src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderSentenceResource.java

```diff
@@ -89,6 +89,17 @@ public HomeDetentionCurfew getLatestHomeDetentionCurfew(@PathVariable("bookingId
         return offenderCurfewService.getLatestHomeDetentionCurfew(bookingId);
     }
 
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "List of HDC status information", response = HomeDetentionCurfew.class, responseContainer = "List"),
```

Comment:
Don't think you need the 200

Reply:
I think you're right, but the other endpoints in this resource all define their 200 responses. Leave in or take out? I'm easy.